liquidcore/home/templates/homepage.html

```diff
@@ -21,54 +21,15 @@ <h1>{{ liquid_title }}</h1>
 
 <p>Welcome to Liquid Investigations.</p>
 
-{% if hoover_app_url %}
-<p>
-  <a href="{{ hoover_app_url }}">Hoover</a> is a search app.
-</p>
-{% endif %}
 
-{% if hypothesis_app_url %}
+{% for app, values in liquid_apps.items%}
```

Comment:
You don't need `app` as a key here and `values` shouldn't be plural.

Simplify this to:

```
{% for app in liquid_apps.values %}
  ... app.enabled and app.adminOnly ...
```